attributes/default.rb

```diff
@@ -8,11 +8,11 @@
 
 #
 # Version of Kafka to install.
-default['kkafka']['version'] = '1.1.0'
+default['kkafka']['version'] = '2.3.0'
 # Version used for properties file
 default['kkafka']['version_properties'] = '1.0'
 # HopsKafkaAuthorizer version
-default['kkafka']['authorizer_version'] = '0.2.0'
+default['kkafka']['authorizer_version'] = '1.0.0-SNAPSHOT'
```

Comment:
@kai-chi do we have a new version of the authorizer?

Reply:
set to 0.3.0-SNAPSHOT